src/pages/Admin/Details/Form/DetailsForm.tsx

```diff
@@ -213,7 +216,11 @@ const DetailsForm: React.FC<DetailsFormProps> = ({ dispatch, editData }) => {
   };
 
   return (
-    <Form onSubmit={handleSubmit(setWorld)} onChange={handleOnChange}>
+    <Form
+      className="DetailsForm"
+      onSubmit={handleSubmit(setWorld)}
+      onChange={handleOnChange}
+    >
       <S.FormInnerWrapper>
```

Comment:
[OOS]

If there aren't that many of these `S.` components, it might be good opportunity to just move those styles in `DetailsForm.scss`

Reply:
fixed in b30532c2bb1f03501255b92b38a51d82471e0261